ui/src/templates/components/CommunityTemplatesActivityLog.tsx

```diff
@@ -105,3 +104,21 @@ export class CommunityTemplatesActivityLog extends PureComponent<Props, State> {
     )
   }
 }
+
+const mstp = (state: AppState) => {
+  return {
+    stacks: state.resources.templates.stacks.filter(
```

Comment:
this could be a getter that protects against things not being ready to go ! if you don't want to make a getter, let's protect agains templates stacks not being there?

Reply:
@ebb-tide i think you might be misunderstanding this. this is just hiding stacks that have been deleted by the user. when you delete a stack, it currently deletes the resources associated with the stack, but leaves the stack itself. `stacks` will always be an array, it's initialized as an empty array by the reducer, and it comes back from the api as an array.

Reply:
we've used getters to filter before.. but getter was just a suggestion :) didn't realize stacks was initialized in reducer, thought it was only defined after fetching. nevermind!